packages/playground/src/components/Playground.tsx

```diff
@@ -0,0 +1,358 @@
+import { useCallback, useState, useRef, useEffect } from 'react';
+
+import { samples } from '../samples';
+import 'react-app-polyfill/ie11';
+import Form, { withTheme } from '@rjsf/core';
+import localValidator from '@rjsf/validator-ajv8';
+import isEqualWith from 'lodash/isEqualWith';
+
+import DemoFrame from './DemoFrame';
```

Comment:
Group all the package imports and local imports together
```suggestion
import { useCallback, useState, useRef, useEffect } from 'react';
import 'react-app-polyfill/ie11';
import Form, { withTheme } from '@rjsf/core';
import localValidator from '@rjsf/validator-ajv8';
import isEqualWith from 'lodash/isEqualWith';

import { samples } from '../samples';
import DemoFrame from './DemoFrame';
```